c7n/resources/vpc.py

```diff
@@ -114,101 +143,130 @@ class FlowLogFilter(Filter):
                 filters:
                   - not:
                     - type: flow-logs
-                      enabled: true
-                      set-op: or
-                      op: equal
-                      # equality operator applies to following keys
-                      traffic-type: all
-                      status: active
-                      log-group: vpc-logs
-
+                      attrs:
+                        - TrafficType: ALL
+                        - FlowLogStatus: ACTIVE
+                        - LogGroupName: vpc-logs
```

Comment:
Wondering if it would be worth keeping legacy and v2 style examples in the docstring here so the docs double as an upgrade guide.